docs/extension/guides/set-wallet-recovery.md

```diff
@@ -0,0 +1,63 @@
+---
+sidebar_label: '🦊 Set Wallet Recovery'
+sidebar_position: 6
+description: 'Extension Support: How to set up wallet recovery'
+---
+
+# Set Wallet Recovery
+
+If you want an autonomous backup for your Universal Profile, you can add an existing wallet as a recovery controller.
+
+## Preparation
+
+To add a new controller account to your Universal Profile, you will need the [Universal Profile Browser Extension](https://chromewebstore.google.com/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn) as well as an [Ethereum wallet with custom network support](../../general/wallet-support.md) installed **in the same [supported browser](../introduction.md#which-browsers-does-the-extension-support)**.
+
+:::tip Wallet Support
+
+The [Wallet Recovery Setup](https://my.universalprofile.cloud/3rd-party/add-recovery) officially supports **MetaMask**as the wallet. If you experience issues with other wallets, please try again using MetaMask.
+
+:::
+
+After installing both extensions in your browser, you can set up the recovery account. Based on the wallet, you can either:
```

Comment:
im not sure about the grammar of "Based on the wallet"

maybe something like "depending on your wallet"

but im not 100% sure and not english native. Just sharing a though here